pkg/cmd/pr/review/review_test.go

```diff
@@ -278,13 +277,13 @@ func TestPRReview_interactive(t *testing.T) {
 		ConfirmFunc:        func(_ string, _ bool) (bool, error) { return true, nil },
 	}
 
-	output, err := runCommand(http, pm, nil, true, "")
+	output, err := runCommand(http, pm, true, "")
 	assert.NoError(t, err)
 	assert.Equal(t, heredoc.Doc(`
 		Got:
 
-		  cool story                                                                  
-		
+		  cool story
+
```

Comment:
The irony ... 🤣

Reply:
In it's current state this file can't be `gofmt` because it moves this whitespace around. Not something I want to fix in this PR though.

Reply:
Yikes, looks like the `gofmt` business requires it 
![Uploading image.png…]()

Reply:
The issue here that the `markdown.Render` includes whitespace which `gofmt` doesn't preserve.

Reply:
Fixed it by pulling back the old code, making the change and saving without formatting.